lib/PDL/PP.pm

```diff
@@ -1390,21 +1391,23 @@ $PDL::PP::deftbl =
       }),
    PDL::PP::Rule::Returns::EmptyString->new("InplaceCode", []),
    PDL::PP::Rule->new("InplaceDocValues",
-     [qw(Name SignatureObj InplaceNormalised)],
+     [qw(Name SignatureObj InplaceNormalised Overload? NoExport?)],
      'doc describing usage inplace',
      sub {
-       my ($name, $sig, $inplace) = @_;
+       my ($name, $sig, $inplace, $ovl, $noexport) = @_;
        my @args = @{ $sig->args_callorder };
        my %inplace_involved = map +($_=>1), my ($in, $out) = @$inplace;
        my $meth_call = $args[0] eq $in;
        @args = grep !$inplace_involved{$_}, @args;
        my @vals = !$meth_call ? () : [
         "\$$in->inplace->$name".(
-           !@args ? '' : "(@{[join ',', map qq{\$$_}, @args]})"
+           !@args ? '' : "(@{[join ', ', map qq{\$$_}, @args]})"
          ).";", []
        ];
-       push @vals, [ "$name(\$$in->inplace".(
-           !@args ? '' : ",@{[join ',', map qq{\$$_}, @args]}"
+       my $op = defined($ovl) ? ref($ovl) ? $ovl->[0] : $ovl : '';
```

Comment:
The way to do this as the "goes" thing:

```
!defined($ovl) ? '' : ref($ovl) ? $ovl->[0] : $ovl;
```

If you have two `?` with no `:` in between, you're not using that style.